### PR TITLE
Make CSV export from admin portal include question text

### PR DESCRIPTION
**Pull request by DoxasticFox**
Closes #74.

### CODE REVIEW

File: app/admin/question_answers.rb [hunk 1]
```diff
@@ -23,6 +23,42 @@
...
+    end
+    column(:user_family_name) do |question_answer|
+      question_answer.user.family_name
+    end
+    column(:consent_question) do |question_answer|
+      question_answer.consent_question.question
+    end
+    column :answer
+    column(:answer_redcap_field) do |question_answer|
+      redcap_response = UploadRedcapDetails.question_answer_to_redcap_response(
```
Comment (DoxasticFox) [inline app/admin/question_answers.rb:45]: All changes to other files in this PR were to provide this `UploadRedcapDetails.question_answer_to_redcap_response` function. `app/lib/upload_redcap_details.rb` (formerly `app/jobs/upload_redcap_details_job.rb`) is where this function was added and contains lots of refactoring. Changes to the tests and `app/models/question_answer.rb` are to accommodate the refactoring.

File: app/models/question_answer.rb [hunk 2]
```diff
@@ -9,10 +9,10 @@ class QuestionAnswer < ApplicationRecord
   before_destroy :destroy_redcap_details
 
   def upload_redcap_details
-    UploadRedcapDetailsJob.perform_now(id)
```
Comment (ammazzaw) [inline app/models/question_answer.rb:12]: My understanding is that the difference in these 2 methods (with/without `_now`) has something to do with serialisation of the record. What has exactly changed?

**Comment by ammazzaw** [comment]
It looks good as far as I can tell.

**Review by ammazzaw — APPROVED** [review]
Looks good. I'd just like to know why the `perform_now()` change (make sure it's the correct thing to do).

**Pull request merged**